src/app/search-page/sort-component/sort.component.scss

```diff
@@ -0,0 +1,6 @@
+mat-form-field::ng-deep.mat-form-field-flex.ng-tns-c76-3 {
```

Comment:
ng-tns-c76-3 looks like a temporary class to me.
Are you sure if your query works after restarting the server?